src/modules/esl-utils/dom/traversing.ts

```diff
@@ -13,6 +13,11 @@ export const createSequenceFinder = (next: (el: Element) => Element | null) => {
   };
 };
 
+/** Checks if element matches passed selector or exact predicate function */
+export const isMatches = (el: Element, matcher?: string | ((el: Element) => boolean)): boolean => {
+  return (typeof matcher === 'string' && el.matches(matcher) || typeof matcher === 'function' && matcher.call(this, el));
```

Comment:
Id recomend to simply call `matcher(el)` or pass context as null, as `this` reference here is not obvious (and `undefined`, by the way)

Also consider to write checkby cases, it will simplify usage, readability and support. Short version in our case will be easely created by any of popular minifiers.

```suggestion
  if (typeof matcher === 'string') return el.matches(matcher);
  if (typeof matcher === 'function') return matcher(el);
  return false;
```
By the way I missed that case with undefined matcher, then the condition in esl-togglable can be simplified
`if (isMatches(this, detail?.ignore)) return;`
And that's, basically, good note for description of utility that i's falcy for emty matcher

PS: just in case you wander why it's good in terms of support do not combine condition cases here I'd have to say two scenarios:
1) readability: as I mentioned above currently it was not obvious that empty matcher is falcy condition, while in suggested alternative it's pretty clear, it may simplify situation for user with read sources if something not mentioned in docs
2) it's very likely that the `isMatchs` may requested in future for array of matches, alternative representation can be extended easily with alow chance of incorrect implementation.